nomad/template_smple.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Was this supposed to have been committed?

Reply:
Nope, removing now